modules/installation-user-infra-machines-advanced-customizing-live-serial-console.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies
+//
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_bare_metal/installing-restricted-networks-bare-metal.adoc
+// * installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc
+
+:_content-type: PROCEDURE
+[id="installation-user-infra-machines-advanced-customizing-live-{boot}-serial-console_{context}"]
+= Modifying a live install {boot-media} to enable the serial console
+
+By default, the serial console is disabled and all output is written to the graphical console. You can enable the serial console with the following procedure.
+
+.Procedure
+
+. Download the `coreos-installer` binary from the link:https://mirror.openshift.com/pub/openshift-v4/clients/coreos-installer/latest/[`coreos-installer` image mirror] page.
+
+ifeval::["{boot-media}" == "ISO image"]
+. Retrieve the {op-system} ISO image from the link:https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/latest/[{op-system} image mirror] page and run the following command to customize the ISO image to enable the serial console to receive output:
```

Comment:
This could be read to imply that it affects the console for the _ISO_ boot, but it only affects the installed system.  The console for the ISO boot can be set with `--live-karg-append console=...`.

Reply:
Good point, i'll add a note around here that addresses this